NBISweden/urdr
# Demonstrate user groups in vacation page

## Type of change

 
## List of changes made

- Update internal db schema with user groups
- Add bash script for obtaining user and group data
- Display on vacation page the  time entries of a group that user belongs to

## Testing
- Delete volumes, rebuild and restart containers
- Regenerate the database.db to get the new schema (see readme)
- Run [backend/sql/update-groups.sh]
- Visit /vacation
